mods/cnc/rules/structures.yaml

```diff
@@ -309,6 +336,18 @@ HAND:
 
 AFLD:
 	Inherits: ^BaseBuilding
+	HitShape:
```

Comment:
With 'ground footprint', do you mean occupied cells or visual footprint of the art?